todolist.js

```diff
@@ -15,6 +15,14 @@
       paragraph.innerText = domElems.inputField.value;
       taskName = domElems.inputField.value;
 
+      const delTaskBtn = document.createElement('button');
+      delTaskBtn.innerHTML = 'Delete';
+      delTaskBtn.addEventListener('click', function () {
+        // localStorage.removeItem();
+        domElems.toDoContainer.removeChild(paragraph);
+        domElems.toDoContainer.removeChild(delTaskBtn);
+      });
+
       taskObj[Date.now()] = {
```

Comment:
store the Date.now() in one variable and use it where we want, because it might get different values

Reply:
Okay, it has been resolved.